### ISSUE TITLE
Provide simple file-list flag

### ISSUE THREAD

**Issue by hzeller** [google/verible#15]
Right now, all  files are provided on the command-line, but it might be useful to have also the concept of a file-list. That way, it is possible to read e.g. an already preprocessed list of files to be further processed by verible (all tools).

The standardized way (common in verilog processing) looks like it is just to have a file that contains a list of command-line options, so not only a list of files, but everything that can be put on the command-line. Common flag name seems to be `-f` ?

Maybe there is already a simple way for the ABSL-flag library to read options from a file ?

**Comment by fangism** [comment]
Behavior like: `tool $(cat flagfile) ...` ?

**Comment by hzeller** [comment]
yes, essentially what the `*.f` files convention do in the EDA verilog world.
